bot/src/index.ts

```diff
@@ -1,9 +1,8 @@
 import { setTimeout as delay } from 'timers/promises';
-import { getNetworkConfigByType } from 'auctions-core/src/constants/NETWORKS';
+import setupNetworks, { getNetworkConfigByType } from 'auctions-core/dist/src/networks';
```

Comment:
Please make sure to never use `dist` folder, since this folder is only updated on the `npm run build` executed inside core folder.

```suggestion
import setupNetworks, { getNetworkConfigByType } from 'auctions-core/src/networks';
```

That's the reason you were stack with outdated code. Here and everywhere else